### PR TITLE
added_support_for_user_resource

### PR DESCRIPTION
**Pull request by SHANDCRUZ**
### Description
Added support for User Resource

### Issues Resolved
[List any issues this PR will resolve. e.g., Fixes #01]

### Check List
- [ ] New functionality includes sanity testing.
  - [ ] All sanity tests pass. (`$ ansible-test sanity`).
  - [ ] All unit tests pass.
- [ ] New functionality has been documented in the README if applicable.
  - [ ] New functionality has been thoroughly documented in the examples (please include helpful comments).

### CODE REVIEW

**Review by AsisBagga — APPROVED** [review]
LGTM

File: roles/oneview_user/README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+## oneview_user
+Manage OneView Users.
+
+#### Synopsis
+ Provides an interface to manage Users. Can create, update, and delete.
+
+#### Requirements (on the host that executes the module)
+  * python >= 3.4.2
+  * hpeOneView >= 5.0.0
```
Comment (yuvirani) [inline roles/oneview_user/README.md:9]: Update oneview version

File: roles/oneview_user_facts/README.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+## oneview_user_facts
+Retrieve the facts about one or more of the OneView Users.
+
+#### Synopsis
+ Retrieve the facts about one or more of the Users from OneView.
+
+#### Requirements (on the host that executes the module)
+  * python >= 3.4.2
+  * hpeOneView >= 5.0.0
```
Comment (yuvirani) [inline roles/oneview_user_facts/README.md:9]: update oneview version

File: roles/oneview_user/README.md (hunk 1 above)
Comment (SHANDCRUZ) [reply roles/oneview_user/README.md:9]: Acutally it supports for >5.0.0

Comment (SHANDCRUZ) [reply roles/oneview_user/README.md:9]: The user resource will support for ov 5.0.0 (from V1200).

**Review by yuvirani — APPROVED** [review]

**Pull request merged**